content/docs/user-guide/programming/gems/code-gems.md

```diff
@@ -43,3 +43,23 @@ The `<Gem>EditorModule` class contains the logic for using the Gem in the Editor
 ### `<Gem>EditorSystemComponent.cpp`, `<Gem>EditorSystemComponent.h`
 
 The `<Gem>EditorSystemComponent` class is the system component class for managing the Editor module. You can use this class to program Editor functionality. For example, if your Gem implements a component or a window that appears in the O3DE Editor.
+
+## Building a Gem
+
+After a Gem is registered to an engine, project, or the global `o3de_manifest.json`, you can build the gem using the same command line tool that you use to [build the engine and projects](/docs/user-guide/build/).  Gems that have been added to a project will be built automatically when the project builds.  While developing a Gem, however, you may want to build it individually to avoid rebuilding the entire project.
+
+In the directory that is appropriate for your [engine installation](/docs/welcome-guide/setup/setup-from-github/building-windows/#build-the-engine), use the `cmake --build` command and specify the build solution and configuration.  Use the `--target` flag to specify the Gem modules that you want to build, `<Gem Name>` for the main Gem module or `<Gem Name>.Editor` for Editor modules.  
+
+The following example shows the command to build _both_ the Editor and main Gem modules for a Gem named `MyGem`:
+
+```cmd
+cmake --build build/windows --target MyGem.Editor MyGem --config profile -- -m
+```
+
+When you build a Gem, the build system will also build any of the Gem's dependencies that are listed in the Gem's `code/CMakeLists.txt` file.
+
+### Building a Gem in Visual Studio
+
+Open the build solution that is appropriate for your [engine installation](/docs/welcome-guide/setup/setup-from-github/building-windows/#build-the-engine), for source engine installations this will be the engine's root directory, for pre-built SDK engines this will be the project's root directory. Then find your Gem's `Code` directory in the Solution Explorer, in most cases it will be located at `Gems\<MyGem>\Code` in the Solution.  This directory contains all of your Gem's build targets, including the main `<MyGem>` and Editor `<MyGem>.Editor` modules.  Ensure that the correct build configuration is selected on the top toolbar. **Right-click** on a module and select **Build** from the menu to build the module.  
```

Comment:
Suggestions explained: 
- Because there are multiple steps, you can organize it into numerical steps.
- For the first step, split it into multiple sentences. 
- Preference for present-tense as appropriate. (E.g. Remove use of "will")
- Added a link explaining build configurations

Question: 
- Do we ever want to build the <Gem>.Static? Or just <Gem> and <Gem>.Editor?
```suggestion
1. Open the Visual Studio build solution that is appropriate for your [engine's build type](/docs/welcome-guide/setup/setup-from-github/building-windows/#build-the-engine). For source engines, this is the engine's root directory, and for pre-built SDK engines, this is the project's root directory. 
1. Find your Gem's `Code` directory in the **Solution Explorer**. In most cases, this directory is located at `Gems\<MyGem>\Code`.  This directory contains all of your Gem's build targets, including the main `<MyGem>` and Editor `<MyGem>.Editor` modules.  
1. Ensure that the correct build configuration is selected in the top toolbar. For information about build configurations, see [Generated build configurations](/docs/user-guide/build/configure-and-build/#generated-build-configurations). 
1. **Right-click** on a module and select **Build** from the menu to build the module.  
```